LayoutTests/platform/glib/TestExpectations

```
@@ -888,6 +888,9 @@ accessibility/dismiss-action-returns-whether-keyevent-handled.html [ Skip ]
 # Different roles
 accessibility/contenteditable-values.html [ Skip ]
 
+# Exposing aria-{row,col}indextext needs to be implemented on glib platforms.
+accessibility/aria-table-indextext.html [ Skip ]
```

@dusek I'm curious if this test (and its failures) should be checked in, rather than skipped? Are they causing a timeout failure?

---

I think in this case, since the test leans so heavily on unimplemented `AccessibilityUIElement` functionality for glib (specifically in `AccessibilityUIElement::stringAttributeValue`), it's OK to skip 🙂 if the test was at least partially meaningful without said missing functionality, we could check in a failure, but I don't think that's the case here.

---

@rahimabdi I simply followed example of other _"Implement aria- attribute"_ commits (specifically [aria-description](https://github.com/WebKit/WebKit/commit/7c414e7250fbc#diff-0408b0d0fdabfde8e9e0d7dc29b91a92455f4e0db9d16db0c7fdc7e2cdc7f8edR211) and [aria-braille*](https://github.com/WebKit/WebKit/commit/8fa0af4fbc0410155ec372865ecc4c869c82defb#diff-0408b0d0fdabfde8e9e0d7dc29b91a92455f4e0db9d16db0c7fdc7e2cdc7f8edR139)) which explicitly added new tests as `TestExpectations` `Skip`s for `glib` (or then `gtk`) and `win` (which now ignores all `accessibility` tests so not needed to add individual tests anymore) when initial support in that commit was only for Apple platforms – does that answer your question please?

---

(And yes I would expect this test to timeout on non-Apple platforms at one point, as it does use a few `await expectAsync...` at the end.)

---

Resolving per @twilco.